docs/readthedocs/source/doc/Nano/Overview/troubshooting.md

```diff
@@ -15,7 +15,7 @@ Please make sure you use context manager provided by ``InferenceOptimizer.get_co
 
 ### ``assert precision in list(self.cur_config['ops'].keys())`` when using ipex quantization with inc on machine with BF16 instruction set
 
-It's known issue for [Intel® Neural Compressor](https://github.com/intel/neural-compressor) that they don't deal with BF16 op well at version 1.13.1 . This will be fixed when next stable version releases.
+It's known issue for [Intel® Neural Compressor](https://github.com/intel/neural-compressor) that they don't deal with BF16 op well at version 1.13.1 . This has been fixed in version 2.0.
```

Comment:
we can tell user to install bigdl-nano >= 2.2 instead of installing inc 2.0 manually